tests/unit/mixins/css-test.js

```diff
@@ -41,35 +37,49 @@ describe('Unit / Mixin / css', function () {
 
   describe('.init()', function () {
     beforeEach(function () {
-      sandbox.stub(component, 'maybeClearClassNameBindings')
+      sandbox.stub(component, 'setClassNameBindings')
       component.init()
     })
 
-    it('should call .maybeClearClassNameBindings()', function () {
-      expect(component.maybeClearClassNameBindings).to.have.callCount(1)
+    it('should call .setClassNameBindings()', function () {
+      expect(component.setClassNameBindings).to.have.callCount(1)
     })
   })
 
-  describe('.maybeClearClassNameBindings()', function () {
+  describe('.setClassNameBindings()', function () {
     describe('when "tagName" is set', function () {
       beforeEach(function () {
         component.set('tagName', 'div')
-        component.maybeClearClassNameBindings()
+        component.setClassNameBindings()
       })
 
-      it('should leave "classNameBindings" alone', function () {
+      it('has classNameBindings set to include "css"', function () {
         expect(component.get('classNameBindings')).to.include('css')
       })
     })
 
     describe('when "tagName" is not blank', function () {
```

Comment:
Probably my typo, but I think this is supposed to say `'when "tagName" is blank'`